distribution/ddf-common/src/main/resources/etc/custom.properties

```diff
@@ -28,46 +28,34 @@ karaf.systemBundlesStartLevel=50
 # You can place any customized configuration here.
 #
 
-#Using equinox until DDF and dependent applications can start appropriately
-#under felix
-#karaf.framework=felix
+#
+# Framework selection properties
+#
+# Using equinox until DDF and dependent applications can start appropriately under felix
 karaf.framework=equinox
 
-# Add console delay, will only open after everything is loaded
+#
+# Add console delay. Set to true if you want the console to start after all other bundles
+#
 karaf.delay.console=true
 
+#
 # Change the default message
+#
 karaf.startup.message=System starting up. Press Enter to open the shell during startup...
 
+#
 # Use BouncyCastle as the default security provider
+#
 org.apache.felix.karaf.security.providers=org.bouncycastle.jce.provider.BouncyCastleProvider
 org.apache.karaf.security.providers=org.bouncycastle.jce.provider.BouncyCastleProvider
 
-# Allow configuration admin to update config files
-felix.fileinstall.enableConfigSave = true
```

Comment:
I just confirmed that the modifications to configurations in the webconsole are saved to the file in `/etc`. I tested that this works for both `.cfg` and `.config` when this `felix.fileinstall.enableConfigSave = true` property isn't included in `custom.properties`.